scripts/export.py

```diff
@@ -141,6 +150,11 @@ def nemo_export(argv):
 
     logging.info("Successfully exported to {}".format(out))
 
+    del model
+
+    if args.runtime_check:
+        verify_runtime(out, input_list, input_dict, input_names, output_names, output_example)
+
```

Comment:
This should really be an implicit part of model export, not an external function. Most people will forget to verify their export in the code when they do model.export().

Reply:
Well I agree in principle, but it should be available externally to be able to check models that won't allow 2 copies on GPU at the same time.